tests/functionality_tests/mission_test.py

```diff
@@ -0,0 +1,54 @@
+from engine.robot import Robot
+from engine.base_station import BaseStation
+from engine.mission import Mission
+from engine.mission import ControlMode
+from engine.database import DataBase
+from engine.robot_state import Robot_State
+from engine.phase import Phase
+
+import math
+
+
+class MissionTest():
```

Comment:
at the top can we add like a one line specification?

"Testing the transitions between mission states" or something?